jupyter-nextflow/nextflow-welcome.html

```diff
@@ -39,21 +39,30 @@ <h1 class=header>Welcome to the Nextflow Workspace</h1>
   </ul>
   <h2 class=header>Get started with Nextflow</h2>
   <p>If you are new to Nextflow, visit <a
-      href=https://www.nextflow.io>nextflow.io</a> for detailed information.
+      href=https://www.nextflow.io>nextflow.io</a> for detailed information. 
+    You can also learn more about using Nextflow in BRH Workspace in the BRH Documentation <a
+      href=https://uc-cdis.github.io/BRH-documentation/01-home/</a>. 
   </p>
   <p>This workspace is set up to run Nextflow workflows in AWS Batch. Your
-    Nextflow configuration must include the Batch queue, IAM role ARN and work
-    directory that were created for you automatically. The sample configuration
+    Nextflow configuration must include the Batch queue (in our config file, 
+    <code>queue</code>), IAM role ARN (<code>jobRole</code>), and work
+    directory (<code>workDir</code>) that were created for you automatically. The sample configuration
```

Comment:
i don't think it's necessary to add more details to this text. it's a high level description of what's in the config file.

in general this file is meant to be short and straight to the point so new users don't skip reading it

Reply:
I found it confusing. I didn't understand what parts mapped to what, and especially when you use the language "must include" and we are telling people later they can edit the config for use -- they should understand which parts go where.